config-connector/solutions/iam/kpt/member-iam/Kptfile

```diff
@@ -0,0 +1,63 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+packageMetadata:
+  shortDescription: sample description
+openAPI:
+  definitions:
+    io.k8s.cli.setters.project-id:
+      x-k8s-cli:
+        setter:
+          name: project-id
+          value: ${PROJECT_ID?}
+          setBy: PLACEHOLDER
+      description: The target project where you want to create service accounts and
+        grant permissions.
+    io.k8s.cli.substitutions.project-id-in-sa-sub:
```

Comment:
Why do we need this separate setter? Can we not set project_id and service_account_name from project-id and service-account-name setters?

Reply:
"project-id-in-sa-sub" is not a setter, but a string substitution. With this subst, we can substitute project id in the service account. You can find more information here: https://googlecontainertools.github.io/kpt/reference/cfg/create-subst/

Reply:
Got it, thanks.